src/NoSignInIAuthClient.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthStatus, BeEvent, BentleyError, ClientRequestContext } from "@bentley/bentleyjs-core";
+import { AccessToken } from "@bentley/itwin-client";
+import { FrontendAuthorizationClient } from "@bentley/frontend-authorization-client";
+
+export class NoSignInIAuthClient implements FrontendAuthorizationClient {
+  public readonly onUserStateChanged: BeEvent<(token: AccessToken | undefined) => void>;
+  protected _accessToken?: AccessToken;
+
+  constructor() {
+    this.onUserStateChanged = new BeEvent();
+  }
+
+  public async signIn(requestContext?: ClientRequestContext): Promise<void> {
+    if (requestContext) {
+      requestContext.enter();
+    }
+    await this.getAccessToken();
+  }
+  public async signOut(requestContext?: ClientRequestContext): Promise<void> {
+    if (requestContext) {
+      requestContext.enter();
+    }
+    this._accessToken = undefined;
+  }
+
+  public get isAuthorized(): boolean {
+    return this.hasSignedIn;
+  }
+
+  public get hasExpired(): boolean {
+    return !this._accessToken;
+  }
+
+  public get hasSignedIn(): boolean {
+    return !!this._accessToken;
+  }
+
+  public async generateTokenString(requestContext?: ClientRequestContext) {
+    if (requestContext) {
+      requestContext.enter();
+    }
+
+    const response = await fetch(`https://prod-imodeldeveloperservices-eus.azurewebsites.net/api/v0/sampleShowcaseUser`);
```

Comment:
Can we make this an environment variable?  It will get bundled at build time then...

Reply:
Yeah I think the URL makes sense to be set as an environment variable - let me make the necessary modifications.